build/parser.mjs

```diff
@@ -1,9 +1,9 @@
 import _ from 'lodash';
 
-import Progress from './progress.mjs';
 import dedupe from './dedupe.mjs';
-import tradable from './tradable.mjs';
+import Progress from './progress.mjs';
 import readJson from './readJson.mjs';
+import tradable from './tradable.mjs';
```

Comment:
not sure why this happened...

Reply:
neither do I, I didn't touch that

Reply:
are you running in VSCode? i'm guessing it did some formatting weirdness

Reply:
passes lint, tho, so i've got no issue